frontend/src/pages/SignUp/index.tsx

```diff
@@ -98,69 +72,94 @@ const SignUp: React.FC = () => {
           <Stack
             component="form"
             direction="column"
-            spacing={1.5}
             sx={(theme) => ({
               marginTop: theme.spacing(2),
               marginBottom: theme.spacing(2),
             })}
-            onSubmit={handleSignUp}
-            noValidate
```

Comment:
Should we keep this too? To prevent the native validation rather than have it show up as a quite a different style from the other custom error messages:

![image](https://github.com/user-attachments/assets/006c8389-da95-453c-9bb7-de849973dc07)